app/services/failsafe_preview_service.rb

```diff
@@ -1,18 +1,18 @@
 # frozen_string_literal: true
 
-class FailsafePreviewService
+class FailsafePreviewService < ApplicationService
   attr_reader :edition
 
   def initialize(edition)
     @edition = edition
   end
 
-  def create_preview
+  def call
```

Comment:
Ah you can see how this change here predicates the need to update the class name.